addons/l10n_mn/data/account.tax.template.csv

```diff
@@ -0,0 +1,6 @@
+id,name,description,type_tax_use,amount_type,amount,chart_template_id:id,sequence,tax_group_id:id
+l10n_mn.tax_sale_vat_10,Sales VAT,10.00%,sale,percent,10,l10n_mn.mn_chart_1,1,l10n_mn.tax_group_no_vat_0
+l10n_mn.tax_purchase_vat_10,Purchase VAT,10.00%,purchase,percent,10,l10n_mn.mn_chart_1,1,l10n_mn.tax_group_10
+l10n_mn.tax_sale_no_vat_0,No VAT,0.00%,sale,percent,0,l10n_mn.mn_chart_1,2,l10n_mn.tax_group_no_vat_0
+l10n_mn.tax_sale_zero_vat_0,Zero VAT,0.00%,sale,percent,0,l10n_mn.mn_chart_1,3,l10n_mn.tax_group_0
+l10n_mn.tax_sale_export_vat_0,Exporting VAT,0.00%,sale,percent,0,l10n_mn.mn_chart_1,4,l10n_mn.tax_group_export_vat_0
```

Comment:
There is a separate csv with all the taxes where almost all the work is done.  You should take that one.

Reply:
ok